src/world/collision_groups.rs

```diff
@@ -0,0 +1,104 @@
+use ncollide::world::CollisionGroups;
+use world::STATIC_GROUP_ID;
+
+/// TODO: better explanations or redirect user to `ncollide` documentation ?!?
+///
+/// Groups of collision used to filter which object collide with which other one.
+/// nphysics use a specific group for its own purposes (i.e. the group of static objects).
+/// The `group 29` is reserved and you cannot use it.
+#[derive(Clone, Debug)]
+pub struct RigidBodyCollisionGroups {
+    collision_groups: CollisionGroups
+}
+
+impl RigidBodyCollisionGroups {
+    /// Creates a new `RigidBodyCollisionGroups` that enables collisions with everything except
+    /// self-collision.
+    #[inline]
+    pub fn new() -> RigidBodyCollisionGroups {
```

Comment:
I think that we should replace this by two constructors: `new_dynamic()` and `new_static()`.
The first one will have its membership to the static group revoked (just as you did) while the second one will have the group `STATIC_GROUP_ID` of its blacklist set to `true` (but will still be a member of the static group).